tests/admin_views/test_nav_sidebar.py

```diff
@@ -35,6 +38,7 @@ def setUpTestData(cls):
         )
 
     def setUp(self):
+        super().setUp()
```

Comment:
This change and `LoggingCaptureMixin` are unnecessary if you will use `assertLogs()`.